src/index.ts

```diff
@@ -292,29 +296,10 @@ export default class Plugin extends AbstractBtpPlugin {
 
   async sendData (buffer: Buffer): Promise<Buffer> {
     const parsedPacket = IlpPacket.deserializeIlpPacket(buffer)
-
-    let destination
-    let isPrepare = false
-    switch (parsedPacket.type) {
-      case IlpPacket.Type.TYPE_ILP_PAYMENT:
-      case IlpPacket.Type.TYPE_ILP_FORWARDED_PAYMENT:
-        destination = parsedPacket.data['account']
-        break
-      case IlpPacket.Type.TYPE_ILP_PREPARE:
-        isPrepare = true
-        destination = parsedPacket.data['destination']
-        if (this._sendPrepare) {
```

Comment:
Was removing `_sendPrepare` intentional? I think xrp-asym-server uses it still.

Reply:
Nice catch! No, it wasn't.

Reply:
https://github.com/interledgerjs/ilp-plugin-mini-accounts/pull/39